protocol/v2/types/crypto.go

```diff
@@ -47,6 +55,43 @@ func VerifyByOperators(s spectypes.Signature, data spectypes.MessageSignature, d
 		return errors.Wrap(err, "could not compute signing root")
 	}
 
+	// verify
+	// if res := sign.FastAggregateVerify(pks, computedRoot[:]); !res {
+	// 	return errors.New("failed to verify signature")
+	// }
```

Comment:
is there a reason to leave that comment?

Reply:
It could be useful but we decided to disable batch verifier so I'm deleting it